sql/postgres/postgrescheck/postgrescheck.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 // NewDataDepend creates new data-depend analyzer.
-func NewDataDepend(*schemahcl.Resource) *datadepend.Analyzer {
+func NewDataDepend() *datadepend.Analyzer {
```

Comment:
This one is needed for other branches I'm working on. Revert, please.

Reply:
Done